gui/ui/src/component/form.rs

```diff
@@ -45,6 +45,19 @@ where
         }
     }
 
+    /// Creates a new [`Form`] that have a disabled input.
+    ///
+    /// It expects:
+    /// - a placeholder
+    /// - the current value
+    pub fn new_disabled(placeholder: &str, value: &Value<String>) -> Self {
```

Comment:
Perhaps instead of adding this method, you could reuse the `new()` method by changing the `on_change` parameter to `Option`.

What do you think, @edouardparis?

Reply:
I had a couple of further thoughts:
1) Unrelated to this PR (perhaps could be a separate commit), we could refactor the `new_trimmed()` and `new_amount_btc()` methods to call `Self::new()` with the modified `on_change` parameter.
2) If these other methods also use `Option`, then you can use the `new_trimmed()` whether or not the replacement txid exists, and the only thing that will change is whether you pass `None` or `Some` to the `on_change` parameter (rather than different methods being called in each case).